src/utils/dispute-utils.js

```diff
@@ -403,32 +440,49 @@ function getRoundPhasesAndTime(courtConfig, round, currentPhase) {
     },
   ]
 
+  // If it's not the last round means that it was appealed and confirm appealed
```

Comment:
I wonder if we can come up with a different wording for "appealed and confirm appealed", like "the dispute reached the next appeal round"?